index.js

```diff
@@ -0,0 +1,463 @@
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-languages-list">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn-block button-primary" id="project">
+              See Project
+            </button>
+          </div>
+          <div class="card">
+            <h1 class="card-heading">Professional Art Printing Data</h1>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-languages-list">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn-block button-primary" id="project">
+              See Project
+            </button>
+          </div>
+          <div class="card">
+            <h1 class="card-heading">Professional Art Printing Data</h1>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-languages-list">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn-block button-primary" id="project">
+              See Project
+            </button>
+          </div>
+          <div class="card">
+            <h1 class="card-heading">Professional Art Printing Data</h1>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-languages-list">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn-block button-primary" id="project">
+              See Project
+            </button>
+          </div>
+          <div class="card">
+            <h1 class="card-heading">Professional Art Printing Data</h1>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-languages-list">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn-block button-primary" id="project">
+              See Project
+            </button>
+          </div>
+        </div>
```

Comment:
@binhussen can you further explain how to make a see project button visible in the card because its not showing me in the figma design. Should I just add the function that if we click anywhere on the card the popup appears?

Reply:
- You can loop through your project array list and call the above prepared function with specific index
```
projects.forEach((ele, i) => {
    showCards(i);
  });
```
check this repo for [more info](https://github.com/binhussen/portfolio)👌